.github/workflows/monitoring.yml

```diff
@@ -0,0 +1,67 @@
+name: Monitoring
+
+on:
+  push:
+  workflow_dispatch:
```

Comment:
Can we also accept arguments to selectively run the monitors, (including all)?

It can be helpful when we're debugging a single monitor and don't want to run the whole matrix again and again.